LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020 Mara contributors
+Copyright (c) 2020-2022 Mara contributors
```

Comment:
I think this isn't needed: at a further company the rule was that the year was the year of the creation of the file.